src/libcfitsio.jl

```diff
@@ -860,6 +871,8 @@ for (a,b) in ((:fits_create_binary_tbl, 2),
         function ($a)(f::FITSFile, numrows::Integer,
                       coldefs::Array{ColumnDef}, extname::String)
```

Comment:
I should have mentioned also here that my only doubt was whether the `coldefs` argument should be checked.

Reply:
Yeah I do think the name and unit at least should be checked because these get written to the file. I'll add those checks (I'm updating the NEWS file anyway).